test/npm-event.js

```diff
@@ -94,4 +94,46 @@ const register = require('../lib/npm-event')
     t.same(job.properties.priority, 1, 'job priority')
     t.end()
   })
+
+  tap.test('accepts packages that are bigger then 1MB', async (t) => {
+    const bigBody = Buffer.alloc(1024 * 1024 * 5).toString()
```

Comment:
it is fine up until `1024 * 1024 * 4.1`. Starting to fail with `1024 * 1024 * 4.2`. Is there some magical number inbetween?

Reply:
"Payload content length greater than maximum allowed: `26 214 400`"

but  1024 * 1024 * 4.2 = `4 404 019.2` ... but (!) after JSON.stringify-ing  the reqPayload is `26 216 594`  the size of the body multiplied by more than 6!